codec.go

```diff
@@ -17,15 +17,24 @@
 package ttrpc
 
 import (
-	"github.com/gogo/protobuf/proto"
 	"github.com/pkg/errors"
+
+	vtproto "github.com/planetscale/vtprotobuf/codec/grpc"
+	gproto "github.com/golang/protobuf/proto"
+	"google.golang.org/grpc/encoding"
 )
 
 type codec struct{}
 
+var proto encoding.Codec
+
+func init() {
+	proto = &vtcodec{vt:&vtproto.Codec{}}
```

Comment:
Where does this get used?

Reply:
The vtcodec type is needed to invoke vtprotobuf's marshal/unmarshal code.